community.json

```diff
@@ -1004,7 +1004,7 @@
         "branch": "master",
         "level": 3,
         "maintained": false,
-        "revision": "a4c309081bd8eedfc7f1865b91b416e83689a79f",
+        "revision": "ef940a1dd1c4b21e42c62f74b244f585980b3050",
         "state": "working",
         "url": "https://github.com/movim/movim_ynh"
```

Comment:
I think you have to change the url for `https://github.com/YunoHost-Apps/movim_ynh` No?

Reply:
Glad you're here !